### PR TITLE
Fix for issue #575

### PR DESCRIPTION
**Pull request by ILMTitan**
Resolve app engine service and version from metadata.

### CODE REVIEW

File: GoogleCloudExtension/ProjectTemplates/GoogleAspNetCoreMvc/Startup.cs [hunk 1]
```diff
@@ -87,5 +79,39 @@ private string GetProjectId()
...
+            }
+            return serviceName;
+        }
+
+        private string GetVersion()
+        {
+            var instance = Google.Api.Gax.Platform.Instance();
+            var versionId =
+                instance?.GaeDetails?.VersionId ??
+                // The source version of the service.See https://cloud.google.com/error-reporting/docs/formatting-error-messages#FIELDS.version.
```
Comment (Deren-Liao) [inline GoogleCloudExtension/ProjectTemplates/GoogleAspNetCoreMvc/Startup.cs]: Move the comment above var versionId ?

File: GoogleCloudExtension/ProjectTemplates/GoogleAspNetCoreMvc/Startup.cs [hunk 2]
```diff
@@ -87,5 +79,39 @@ private string GetProjectId()
             }
             return projectId;
         }
+
+        private string GetServiceName()
+        {
+            var instance = Google.Api.Gax.Platform.Instance();
+            var serviceName =
+                instance?.GaeDetails?.ServiceId ??
+                // An identifier of the service. See https://cloud.google.com/error-reporting/docs/formatting-error-messages#FIELDS.service.
```
Comment (Deren-Liao) [inline GoogleCloudExtension/ProjectTemplates/GoogleAspNetCoreMvc/Startup.cs]: Nit: Move the comment up?

File: GoogleCloudExtension/ProjectTemplates/GoogleAspNetCoreMvc/Startup.cs (hunk 1 above)
Comment (ILMTitan) [inline GoogleCloudExtension/ProjectTemplates/GoogleAspNetCoreMvc/Startup.cs]: Done.

File: GoogleCloudExtension/ProjectTemplates/GoogleAspNetCoreMvc/Startup.cs (hunk 2 above)
Comment (ILMTitan) [inline GoogleCloudExtension/ProjectTemplates/GoogleAspNetCoreMvc/Startup.cs]: Done.

**Pull request merged**